javascripts/background_scripts/modal_button.js

```diff
@@ -14,6 +14,9 @@
  *
  */
 
+
+  /*global chrome:false, ls:true, readingProcess:false, */
```

Comment:
@smcgregor  Yes. true as a property represents that the variable can be edited by the code from current file. False means the code in current file can read that variable but can't make any change to it.